etc/nginx/nginx.conf

```diff
@@ -16,8 +16,12 @@ http {
 	include               /etc/nginx/mime.types;
 	default_type          application/octet-stream;
 
-	access_log            /var/log/nginx/access.log;
-	error_log             /var/log/nginx/error.log;
+	log_format rt_cache '$remote_addr - $upstream_cache_status [$time_local]  '
+											'"$request" $status $body_bytes_sent '
```

Comment:
The indentation is correct in the editor. It's messed up here since GitHub uses 4 spaces for a tab.

![image](https://user-images.githubusercontent.com/306968/67546707-4c5f8d00-f6fd-11e9-8ca7-33c0eecd7218.png)

Reply:
I'll just replace all tabs with spaces in a separate commit.